firmware/application/apps/ui_recon.cpp

```diff
@@ -1339,7 +1339,7 @@ size_t ReconView::change_mode(freqman_index_t new_mod) {
     field_bw.on_change = [this](size_t, OptionsField::value_t) {};
     recon_stop_recording();
     remove_child(record_view.get());
-
+    record_view.reset();
```

Comment:
Something looks off here. It's missing the else branch I used to have... Let me look at the history.

Reply:
NVM, just the diff view being "helpful" by collapsing 3 lines. *smh
Anyway, that reset doesn't do anything. It's being reassigned in both branches here and assignment is the same as `reset` per the docs.

Reply:
Actually double NVM (you're right) --- if the RecordView is doing that message registration in its ctor (or in-class initialization) - I think unique_ptr doesn't delete until the new instance is created (for exception safety). So that's a good thing to keep in mind.